plugins/appInitPlugin.ts

```diff
@@ -4,6 +4,9 @@ const appInitPlugin: Plugin = (context) => {
   if (context.store.state.user.serverUrl) {
     context.$axios.setBaseURL(context.store.state.user.serverUrl);
   }
+  if (context.store.state.servers.ServerAddress) {
```

Comment:
This doesn't seem right?

Reply:
Good catch, this isn't needed since we're using the server address from the user store